features/posting/screens/src/main/java/social/plasma/features/posting/screens/ComposePostUiEvent.kt

```diff
@@ -1,10 +1,14 @@
 package social.plasma.features.posting.screens
 
+import androidx.compose.ui.text.input.TextFieldValue
 import com.slack.circuit.CircuitUiEvent
+import social.plasma.models.TagSuggestion
 
 
 sealed interface ComposePostUiEvent : CircuitUiEvent {
-    data class OnNoteChange(val content: String) : ComposePostUiEvent
+    data class OnNoteChange(val content: TextFieldValue) : ComposePostUiEvent
```

Comment:
This feels like a code smell, but I understand the use of it to not have to write an extra abstraction.